concord-ui/src/components/systemstatstable.js

```diff
@@ -139,7 +139,7 @@ export class SystemStatsTable extends Component {
       <div>
         <SystemStatsTable />
         <Typography style={{ fontSize: '30px' }} use='subtitle1' tag='h2'>
-          {deadLockedPhrases.length} Deadlocked Phrases
+          {deadLockedPhrases.length.toLocaleString()} Deadlocked Phrases
```

Comment:
What's this for?

Reply:
will format the string, e.g. 1000 -> 1,000